Modules/IO/ImageBase/include/itkImageIOBase.h

```diff
@@ -577,39 +579,38 @@ class ITKIOImageBase_EXPORT ImageIOBase : public LightProcessObject
     this->SetPixelType(IOPixelEnum::RGBA);
     this->SetComponentType(MapPixelType<TPixel>::CType);
   }
-  template <typename TPixel, unsigned VLength>
+  template <unsigned VLength>
   void
-  SetPixelTypeInfo(const Vector<TPixel, VLength> *)
+  SetPixelTypeInfo(const Offset<VLength> *)
```

Comment:
It is hard to see what are changes and what are just moving definitions around. Can reordering of definitions be done in a separate (first) commit?

Reply:
It could be easier to review by splitting the commits, but it is easier to write without bugs by ordering as part of the process to find what is wrong and missing in the first place. The order and content was based on the `PixelEnum` contents. The changes are summarized in the commit message, which is less of a burden to review than splitting the commits.